### DISCUSSION TITLE
Get method query single parameter used twice

### DISCUSSION THREAD

**Discussion by basavarajdodamani** [strapi/strapi#7997] [Legacy Topics]
Hello,
I see difference in the behaviour of two of the variations of GET URL on a collection:

http://localhost:1337/api-details?api_name=GetStudentDetails&api_name=GetEducationDetails
Returns documents/rows having both GetStudentDetails and GetEducationDetails

http://localhost:1337/api-details?_q=details&_q=student
Returns documents/rows available in the collection.

In the second part, i know that _q tries to find the content in all columns/fields of a collection. In my case I wanted to find more than one value across all columns/fields, in one invocation. Is this possible?

![image](https://user-images.githubusercontent.com/69140968/93965157-0254b080-fd7f-11ea-8891-04f68b7a9c22.png)

Kind Regards,
Basu

**Comment by alexandrebodin** [comment]
Hi, you can use a or operator :) https://strapi.io/documentation/v3.x/content-api/parameters.html#complex-queries

**Reply by basavarajdodamani** [reply]
Thanks @alexandrebodin  You mean i can use or operator on **_q** or you are suggesting to implement a custom query?

**Reply by alexandrebodin** [reply]
Sorry If I wasn't clear. I meant you can run _or queries to do want you want with the api-name=X _q is only available as a unique query parameter though

**Reply by basavarajdodamani** [reply]
not a problem Alex. Yeah i can apply operators on columns/fields in the query. But i wanted for _q parameter. My main objective is to find matching records based on two input values that may be available in any of the columns of the collection. For example, 
http://localhost:1337/api-details?_q=details&_q=class,
Expectation to return records based on **class** matching on api_name and short_descr columns (and any other columns having it) plus **details** matching records.
Should return below
![image](https://user-images.githubusercontent.com/69140968/93994485-90e02680-fdad-11ea-972d-77f8bbd432bf.png)

**Comment by alexandrebodin** [comment]
If I understand your usecase, the closest you can achieve is using something like:

```
qs.stringify({
_q: "details",
_where: {
  _or: [
    { api-name_containes: "Class" },
    { short_description_containes: "Class" },

 ]
}})
```
